src/components/shared/Buttons/index.tsx

```diff
@@ -0,0 +1,49 @@
+import classnames from 'classnames';
+import { ButtonHTMLAttributes } from 'react';
+
+export enum ButtonVariants {
+  Primary = 'primary',
+  Outline = 'outline',
+}
+
+interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  variant: ButtonVariants;
+  hasIcon?: boolean;
+  isLarge?: boolean;
+  notBold?: boolean;
+  isMedium?: boolean;
+  isSmall?: boolean;
+}
+
+const Button = ({
+  variant,
+  children,
+  hasIcon,
+  isLarge,
+  isMedium,
+  isSmall,
```

Comment:
These three boolean are exclusive?  A button can be large and medium at the same time? Otherwise maybe is a best approach to use a type instead 3 booleans